Dockerfile

```diff
@@ -27,7 +27,7 @@ ENV HOST ""
 WORKDIR /data
 
 RUN apk add --no-cache bash
-COPY ./wait-for-it.sh /usr/local/bin
+#COPY ./wait-for-it.sh /usr/local/bin
```

Comment:
wait-for-it.sh is used. so this comment out is mistake?

Reply:
The wait-for-it.sh file is moved to the bin folder, which is copied in the beginning of the Dockerfile execution.
just forgot to delete this comment.